test/index.js

```diff
@@ -2369,6 +2369,31 @@ describe('expect()', () => {
             });
         });
     });
+
+    it('handles cases where thrownAt() cannot parse the error', () => {
+
+        const captureStackTrace = Error.captureStackTrace;
+
+        Error.captureStackTrace = (error) => {
+
+            error.stack = 5;
+        };
+
+        try {
+            Code.expect(1).to.equal(2);
+        }
+        catch (err) {
+            var exception = err;
```

Comment:
I wouldn't use `var` and rather declare a `let` variable next to `captureStackTrace` but it's just nitpicking.

Reply:
That's part of hapi's [custom ESLint setup](https://github.com/hapijs/rule-no-var). It's used in other places in this file, and across other hapi modules.

Reply:
Alright, my bad. I didn't know about this rule. I didn't check in the file if it was a recurrent pattern.